lib/adapters/sequelize.js

```diff
@@ -301,34 +411,26 @@ module.exports = Bone => {
       throw new Error('unimplemented');
     }
 
-    static scope() {
-      throw new Error('unimplemented');
-    }
-
     static async sum(attribute, options = {}) {
       let spell = super.find(options.where).$sum(attribute);
-      if (options.paranoid === false) spell = spell.unscoped;
+      if (options.paranoid === false) spell = spell.unparanoid;
       return await spell;
     }
 
     // EXISTS
     // static async sync() {}
 
-    static truncate() {
-      throw new Error('unimplemented');
-    }
-
-    static unscoped() {
-      const spell = this._find();
-      spell.scopes = [];
-      return spell;
+    // TODO or TRUNCATE TABLE tableName ?
+    static truncate(options) {
+      const opt = options || {};
+      return this.remove({}, true, opt);
```

Comment:
I think a specific `TRUNCATE` sql is necessary. The actual query can be implemented at the driver schema.